cli/tests/pcluster/config/test_validators.py

```diff
@@ -483,6 +484,37 @@ def test_raid_validators(mocker, section_dict, expected_message):
     utils.assert_param_validator(mocker, config_parser_dict, expected_message)
 
 
+def test_kms_key_validator(mocker, boto3_stubber):
```

Comment:
I approved the patch but I'd add a test for a failure case (kms key not present).

Reply:
I'll add it in a seperate PR